ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/table-export/table-export.tsx

```diff
@@ -66,6 +75,17 @@ export default hot(module)((props: Props) => {
         label="Export"
         onChange={handleExportSizeChange}
       />
+      <div hidden={exportSize !== 'custom'}>
```

Comment:
❗️ Instead of hiding the component we should update this to instead not render the input at all when unnecessary.

Also, we should ` import Text from '../../container/input-wrappers/text'` and use that instead of input.

Reply:
Updated to only render when necessary, no longer hiding.

Also changed from `input` to `Number`